.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
```

Comment:
Specifically, I'm referring to the `assignees` column in the table. I read the intention with this is to allow bug / feature request reports to assign the task to someone.

Reply:
You are right, there is no need for contributor to identify who is it assign to. I agree to dropping it as well.

Reply:
That also brings up the labels field, will Github default labels be sufficient, or we would like to define our own labels?

Reply:
I think we should start with the default and add / edit as needed. Not opposed to doing that now if there are certain labels that make sense

Reply:
Definitely, we can always update these templates if we found that it doesn't fit our need.